frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -30,6 +34,19 @@ export const EmbedHomepage = () => {
     getDocsUrl(state, { page: "embedding/static-embedding" }),
   );
 
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  const defaultTab = useMemo(() => {
+    // we want to show the interactive tab for EE builds
+    // unless it's a starter cloud plan, which is EE build but doesn't have interactive embedding
+    if (isEEBuild()) {
+      return plan === "starter" ? "static" : "interactive";
```

Comment:
Small note: we want to show interactive even if there is no license active, the content of the card will prompt to activate the license. That's why I'm checking if it's a ee build and not if the plan is ee on line 44